docs/ck-tutorial.rst

```diff
@@ -490,6 +490,52 @@ shock tube at Stanford.
     ...
 
 
+Converting from ReSpecTh
+------------------------
+
+PyKED provides converter functions from (and to) the ReSpecTh XML format
+[Varga2015a]_, [Varga2015b]_, though ChemKED files created in this manner may
+require manual edits to fully satisfy our schema.
+
+Given a ReSpecTh XML file ``file.xml``, a corresponding ChemKED file can be created
+either from the command line
+
+.. code-block:: bash
+
+    python converter -i file.xml -o file.yaml
```

Comment:
OK, I thought that by defining the entry point, you don't need the module: 
```
 entry_points = {
        'console_scripts': ['converter=pyked.converters:main'],
    }
```
But, I can make the title more specific—does `respth2ck` imply a direction?